src/aliceVision/sfmData/Landmark.hpp

```diff
@@ -63,6 +63,7 @@ struct Landmark
   feature::EImageDescriberType descType = feature::EImageDescriberType::UNINITIALIZED;
   Observations observations;
   image::RGBColor rgb = image::WHITE;    //!> the color associated to the point
+  IndexT referenceView = UndefinedIndexT;
```

Comment:
I'm not sure what's the meaning of this variable.
Could you add documentation?
Also, I don't see any place where this value is set.